src/main/java/uk/gov/mca/beacons/api/domain/LegacyBeacon.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.mca.beacons.api.domain;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class LegacyBeacon {
+
+  private UUID id;
+  private Map<String, Object> beacon;
+  private Map<String, Object> emergencyContact;
+  private List<Map<String, Object>> uses;
+  private List<Map<String, Object>> owners;
```

Comment:
Split this out into:
- Owner
- Secondary owners
And have the ETL process send across the main owner and secondary owners